aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -39,6 +39,19 @@ void fill_conv_desc(MPSGraphConvolution2DOpDescriptor* descriptor_,
   descriptor_.groups = groups;
 }
 
+static
+MPSShape* get_mps_conv_shape(bool is_channels_last, const Tensor& tensor) {
```

Comment:
Nit: usually parameters to the function are specified after the tensor.
Also, is't memory layout a property of tensor? I.e. function should just take tensor as input and check memory layout internally?
```suggestion
MPSShape* get_mps_conv_shape(const Tensor& tensor, bool is_channels_last) {
```